File: Sem- 3/Data Structure/Array/OddEven.java
import java.util.Scanner;
public class OddEven{
	public static void main(String[] args){
		Scanner sc=new Scanner(System.in);
		System.out.println("Odd or Even");
		System.out.print("Enter any number: ");
		int n=sc.nextInt();
		if(n%2==0){
			System.out.println("Even");
		}else{
			System.out.println("Odd");
		}
	}
}

File: Sem-1/C/If Else/oddeven.c
#include<stdio.h>
void main(){
	printf("Even or Odd");
	printf("\nEnter any number: ");
	int a;
	scanf("%d" , &a);
	if(a%2==0){
		printf("Even Number");
	}
	else{
	    printf("Odd Number");
	}
}
